### PR TITLE
Updating to Graph SDK 5.

### PR DESCRIPTION
**Pull request by jmprieur**

### CODE REVIEW

File: AppModelv2-WebApp-OpenIDConnect-DotNet/Web.config [hunk 1]
```diff
@@ -55,51 +51,55 @@
...
-			<dependentAssembly>
-				<assemblyIdentity name="Microsoft.Owin" publicKeyToken="31BF3856AD364E35" culture="neutral"/>
-				<bindingRedirect oldVersion="0.0.0.0-4.2.2.0" newVersion="4.2.2.0"/>
+				<assemblyIdentity name="Microsoft.Kiota.Abstractions" publicKeyToken="31BF3856AD364E35" culture="neutral"/>
+				<bindingRedirect oldVersion="0.0.0.0-1.1.1.0" newVersion="1.1.1.0"/>
 			</dependentAssembly>
 			<dependentAssembly>
 				<assemblyIdentity name="Microsoft.IdentityModel.Tokens" publicKeyToken="31BF3856AD364E35" culture="neutral"/>
-				<bindingRedirect oldVersion="0.0.0.0-6.29.0.0" newVersion="6.29.0.0"/>
+				<bindingRedirect oldVersion="0.0.0.0-6.30.0.0" newVersion="6.30.0.0"/>
```
Comment (jennyf19) [inline AppModelv2-WebApp-OpenIDConnect-DotNet/Web.config:74]: I think 6.31.0 is latest wilson

**Review by jennyf19 — APPROVED** [review]
:shipit:

File: AppModelv2-WebApp-OpenIDConnect-DotNet/Web.config (hunk 1 above)
Comment (jmprieur) [reply AppModelv2-WebApp-OpenIDConnect-DotNet/Web.config:74]: Thanks.
I didn't force it to latest. It came with Id.Web 2.12.2

**Comment by wizofaus** [comment]
How do we rewrite code like `.Request().WithAppOnly().GetAsync()` with the new MS Graph?

**Pull request merged**